### DISCUSSION TITLE
Make a Wiki section about Logs

### DISCUSSION THREAD

**Discussion by Francesco146** [anddea/revanced-patches#526] [Ideas]
a section with a brief explanation of how to get logs and how to share them without personal information.

ref to: #523

**Comment by Francesco146** [comment]
i could do this in case @anddea is busy

**Comment by Francesco146** [comment]
also i suggest creating some labels just like RVX issue center.

**Comment by anddea** [comment]
Not sure why we need this. Doesn't inotia's repo has docs?

**Reply by Francesco146** [reply]
not for the logs gathering

**Reply by anddea** [reply]
People don't read. Before I pushed fix release, I received 3 same issues, I also received 3 same requests for search, was it hard to search other issues, don't think so.

**Reply by anddea** [reply]
Although it might be nice to have good docs/wikis, at the same time it's pointless since people don't read.
